webdriver-spec.html

```diff
@@ -2850,12 +2853,15 @@
   or for the <a>session page load timeout</a> milliseconds to pass,
   whichever occurs sooner.
 
-  <p class=issue>This doesn’t actually wait for the page to load
-   if traversing the session history kicks off a navigation.
+  <li><p>If traversing the session history causes <a>navigation</a>:
```

Comment:
True. If we can agree on using the `page load strategy` to determine when to process the next command then I'll remove this text, and the matching one below.